assets/airports/lszh.json

```diff
@@ -0,0 +1,1812 @@
+                ["tra", 4]
+            ]
+        },
+                {
+            "origin": "LSZH",
+            "destination": "",
+            "category": "departure",
+            "route": "LSZH.VEBIT3S.VEBIT..BALIR..LUMEL..TORPA..LUL",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.3574,
+            "airlines": [
+                ["aal/long", 5],
+                ["aca/long", 3],
+                ["dal/long", 5],
+                ["edw/long", 15],
+                ["swr/long", 35],
+                ["ual/long", 7]
+            ]
+        },
+        {
+            "origin": "LSZH",
+            "destination": "",
+            "category": "departure",
+            "route": "LSZH.VEBIT3W.VEBIT..BALIR..LUMEL..TORPA..REKLA",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.926478,
+            "airlines": [
+                ["afr", 11],
+                ["baw", 18],
+                ["cfe", 9],
+                ["ein", 7],
+                ["ewg", 2],
+                ["ezy", 10],
+                ["ice", 3],
+                ["klm", 12],
+                ["hb/fastGA", 3],
+                ["swr", 42],
+                ["tra", 4]
+            ]
+        },
+        {
+            "origin": "LSZH",
+            "destination": "",
+            "category": "departure",
+            "route": "LSZH.VEBIT3S.VEBIT..BALIR..LUMEL..TORPA..REKLA",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.4535,
+            "airlines": [
+                ["aal/long", 5],
+                ["aca/long", 3],
+                ["dal/long", 5],
+                ["edw/long", 15],
+                ["swr/long", 20],
+                ["ual/long", 7]
+            ]
+        },
+        {
+            "origin": "LSZH",
+            "destination": "",
+            "category": "departure",
+            "route": "LSZH.VEBIT3W.VEBIT..ROTOS..BADEP..ULMES",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 3.105,
+            "airlines": [
+                ["edw", 25],
+                ["ezy", 1],
+                ["gsw", 6],
+                ["hb/fastGA", 2],
+                ["swr", 51],
+                ["tap", 10],
+                ["vlg", 25]
+            ]
+        },
+        {
+            "origin": "LSZH",
+            "destination": "",
+            "category": "departure",
+            "route": "LSZH.VEBIT3S.VEBIT..ROTOS..BADEP..ULMES",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.36,
+            "airlines": [
+                ["swr/long", 15]
+            ]
+        },
+        {
+            "origin": "LSZH",
+            "destination": "",
+            "category": "departure",
+            "route": "LSZH.VEBIT.VEBIT..OBEDU..OMIDO..ARTAG..GERSA..SOSON",

```

Comment:
Two things here, `VEBIT` is not a SID, so this causes some errors. Also, if it was a SID, the entry should always include the runway.

Reply:
Also, if you're interested, these sorts of routes seem like good opportunities to make use of airways! `VEBIT.T53.GERSA.N850.SOSON`